WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift

```diff
@@ -161,6 +161,31 @@ private extension BillingInformationViewController {
     func displayMessageComposerIfPossible(from: UIViewController) {
         messageComposer.displayMessageComposerIfPossible(order: order, from: from)
     }
+
+    /// Create an action sheet that offers the option to copy the email address
+    ///
+    func displayEmailCopyAlert(from: UIViewController) {
+        guard let sourceView = from.view, let email = order.billingAddress?.email else {
+            return
+        }
+
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .primary
```

Comment:
could you update the color to `.text` from the Dark mode color updates on `release/3.2`?

Reply:
Normally I would use that color, except this is an action sheet button. I used [this Figma design](https://www.figma.com/file/S62xbAwtgoMgax4GcuFjKS/Dark%2FLight-modes?node-id=170%3A5076) as a reference.